docs-site/src/pages/index.tsx

```diff
@@ -6,25 +6,13 @@ import { graphql, PageProps } from 'gatsby';
 
 export default function Index({ data }: PageProps<Queries.MarkdownPageQuery>) {
   const { markdownRemark } = data;
-  const { html } = markdownRemark!!;
+  const { frontmatter, html } = markdownRemark!!;
 
   return (
     <Layout currentPath="/">
-      <ArticleHeader title="What is EOP" section="Introduction" />
+      <ArticleHeader title={frontmatter!!.title!!} />
       <Prose>
-        <div>
-          <p className="text-center font-bold italic">
-            The Environmental Outcomes Platform (EOP) is an end-to-solution for
-            acquiring, managing, and interacting with environmental information
-            to empower council services both internally and externally.
-          </p>
-          <p className="text-center font-bold italic">
-            The vision is to have an environment focused platform that has
-            outcomes at heart.
-          </p>
-
-          <div dangerouslySetInnerHTML={{ __html: html!! }} />
-        </div>
+        <div dangerouslySetInnerHTML={{ __html: html!! }} />
```

Comment:
I was trying to get rid of this and just use the regular markdown generator, but couldn't get it to work ... also only spent 10 minutes on it, and gave up when it didn't work first time